requirements.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+dnf install -y dnf-plugins-core make golang-github-cpuguy83-go-md2man python-pexpect telnet
+dnf copr enable -y phracek/meta-test-family-devel
+dnf install -y --enablerepo=updates-testing meta-test-family distgen
```

Comment:
it is really need to have there this `--enablerepo=updates-testing` I know that it was because of __workaround__ for bad library for docker file parsing in current fedora26 if I remember correctly. And maybe in this case __copr enable__ there could be as first line and then just one `dnf install -y ...` command

Reply:
I will try it. Make sense to me

Reply:
dnf-plugins-core is needed for copr and we would like to have the lastest packages and also packages mentioned in testinst.